### PR TITLE
Update resources.md

### PR DESCRIPTION
**Pull request by franceme**
Trimmed the following links from the resources page
* Officers link
* Nomination form for Grad Council
* Officers
* Organizing an event
* CS Grad student directory (listed within another link - gpc.cs)
* The handbook url
* The GTA form

### CODE REVIEW

File: csgc-jekyll-src/resources.md [hunk 1]
```diff
@@ -6,28 +6,21 @@ permalink: /resources/
 
 ## Points of contact 
 
-* Please don't hesitate to [email any of us](/Officers/)!
```
Comment (ayaankazerouni) [inline csgc-jekyll-src/resources.md]: Maybe we should keep this link, since it's a dedicated "points of contact" section.

Comment (tchlux) [reply csgc-jekyll-src/resources.md]: Either that, or call it a "Social Media" section and include a link to the Slack channels that Steve started.

Comment (tchlux) [reply csgc-jekyll-src/resources.md]: On further thought, I suggest we make this a "Social Media" section and include a link to the Slack page.

Comment (ayaankazerouni) [reply csgc-jekyll-src/resources.md]: Good suggestions! Myles, let's go ahead and do that.

**Pull request merged**